routers/issueRouter.js

```diff
@@ -42,4 +42,16 @@ router.post( '/', jsonParser, function( req, res ){
 	});
 });
 
-module.exports = router;
+// PUT issue, updating a current issue
+router.put( '/', jsonParser, function( req, res){
+	var issue = req.body;
+	db.findByIdAndUpdate(issue._id, { name: issue.name, description: issue.description, location: { lat: issue.location[lat], long: issue.location[long]}, priority: issue.priority, open: issue.open, reportedDate: issue.reportedDate, resolvedDate: issue.resolvedDate}, function( err, doc ){
```

Comment:
I think findByIdandUpdate only needs the fields that will be updated. So could try something like:
`var id = issue._id`
`delete issue._id; //this removes the _id field from the issue object`
`db.findByIdAndUpdate( id, issue, function( ... ){ ... } );`
In the future. That way the client only passes in attributes of the issue that changed.